### PR TITLE
[ESLint] autofix 'interface'

### PR DESCRIPTION
**Pull request by luclu**
- only contains non-breaking fixes.
- I validated diff of every single change
- tested Mist

### CODE REVIEW

File: interface/client/templates/layout/sidebar.js [hunk 1]
```diff
@@ -108,36 +110,36 @@ Template['layout_sidebar'].helpers({
...
+    'click nav ul.sub-menu button': function (e, template) {
         var tabId = $(e.currentTarget).parent().parents('li').data('tab-id');
-        var webview = $('webview[data-id="'+ tabId +'"]')[0];
+        var webview = $('webview[data-id="' + tabId + '"]')[0];
 
         // browser
-        if(tabId === 'browser') {
-            webviewLoadStart.call(webview, tabId, {newURL: this.url, type: 'side-bar-click', preventDefault: function(){}});
+        if (tabId === 'browser') {
+            webviewLoadStart.call(webview, tabId, { newURL: this.url, type: 'side-bar-click', preventDefault() {} });
```
Comment (frozeman) [inline interface/client/templates/layout/sidebar.js]: Thats ES6, which may not work in meteor frontend right now.

File: interface/client/templates/layout/webviews.js [hunk 2]
```diff
@@ -11,13 +11,13 @@ The main section template
 @constructor
 */
 
-Template['layout_webviews'].helpers({
+Template.layout_webviews.helpers({
```
Comment (frozeman) [inline interface/client/templates/layout/webviews.js]: The way i wrote it, is for the reason of readability. So i want to keep them, as `Template['layout_webviews']`

Comment (luclu) [inline interface/client/templates/layout/webviews.js]: Reverted this change. (rule: `dot-notation`)

**Comment by evertonfraga** [comment]
Fixed the ES6 issues in /interface/client. Now both Travis and I can build Mist properly.

**Comment by evertonfraga** [comment]
@luclu when do you plan to merge this?

@frozeman i've fixed that ES6 syntax on `/interface` dir.

**Comment by frozeman** [comment]
Please resolve conflicts

**Comment by luclu** [comment]
superseeded

**Pull request closed without merging**